setup.py

```diff
@@ -654,14 +654,15 @@ def find_file_in_path(filename):
 
 
 args['data_files'] = [
+    ('share/metainfo', ['org.musicbrainz.Picard.appdata.xml']),
     ('share/icons/hicolor/16x16/apps', ['resources/images/16x16/picard.png']),
     ('share/icons/hicolor/24x24/apps', ['resources/images/24x24/picard.png']),
     ('share/icons/hicolor/32x32/apps', ['resources/images/32x32/picard.png']),
     ('share/icons/hicolor/48x48/apps', ['resources/images/48x48/picard.png']),
     ('share/icons/hicolor/128x128/apps', ['resources/images/128x128/picard.png']),
     ('share/icons/hicolor/256x256/apps', ['resources/images/256x256/picard.png']),
     ('share/icons/hicolor/scalable/apps', ['resources/img-src/picard.svg']),
```

Comment:
All these icons also need to be renamed to `org.musicbrainz.Picard.*`.

Which means you must also modify the `Icon=` key in the desktop file. :wink:

Reply:
Should I also rename all icon png files in resources manually or they are auto-generated and I do not need to change their name. @bochecha

Reply:
I see no indication that they are auto-generated, and they are even committed in Git.

Also, I just found that if you rename the image files, you also need to rename them there: https://github.com/metabrainz/picard/blob/master/resources/picard.qrc

To identify which one you need to rename, it's all the ones that get installed by `setup.py` in `${prefix}/share/icons/hicolor/${SIZE}/apps`.

Reply:
The images need to be refactored [here](https://github.com/metabrainz/picard/blob/5241c32c49e50aae49c2653f29713aea21ef6399/picard/ui/mainwindow.py) too.